### PR TITLE
Convert genomeinfodb

### PR DESCRIPTION
**Pull request by villafup**
@Bioconductor/sweave2rmd Please Reviews

### CODE REVIEW

File: vignettes/GenomeInfoDb.Rmd [hunk 1]
```diff
@@ -0,0 +1,322 @@
+---
+title: "An Introduction to [*GenomeInfoDb*](http://bioconductor.org/packages/GenomeInfoDb)"
+date: "Modified 26 June, 2020.  Compiled  `r format(Sys.time(), '%B %d, %Y')`"
```
Comment (jwokaty) [inline vignettes/GenomeInfoDb.Rmd:3]: The `author` line was left out.

File: vignettes/GenomeInfoDb.Rmd [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+## keep only the seqlevels you want (say autosomes)
+auto <- extractSeqlevelsByGroup(species="Homo sapiens", style="NCBI",
+                                group="auto")
+x <- keepSeqlevels(x,auto)
+```
+
+# Session Information
+
+Here is the output of on the system on which this document was compiled:
+\<\<sessionInfo, results='asis', eval=TRUE\>\>= toLatex(sessionInfo())
```
Comment (LiNk-NY) [inline vignettes/GenomeInfoDb.Rmd:317]: This code is replaced by `sessionInfo()` below. Remove 317 and 318.

**Comment by hpages** [comment]
Hi @villafup @jwokaty @LiNk-NY What's the status of this PR? Is someone going to review it, or has it been superseded by another PR? Thanks

**Comment by jwokaty** [comment]
@hpages I'm sorry, but I'm very, very, very behind on reviews.

**Comment by villafup** [comment]
Hi @hpages.  I thought I had finished this one.  Sorry but I'm still pretty new to github.  It looks like I might have some write access issues?  In any case I will discuss with @jwokaty  tomorrow.

**Comment by hpages** [comment]
@jwokaty It's ok no worries, I didn't mean to put pressure. I was just going thru the open PRs for **GenomeInfoDb** trying to spot those that maybe had been abandoned or superseded, in which case I would close. Thanks!

**Comment by hpages** [comment]
Any update on this? @villafup I'm still not sure I understand why you have submitted 3 PRs (#38, #39, #41) that all seem to be about converting the GenomeInfoDb.Rnw vignette to Rmd. If some of them are obsolete/superseded/abandonned then they should be closed. Thanks!

File: vignettes/GenomeInfoDb.Rmd (hunk 2 above)
Comment (LiNk-NY) [reply vignettes/GenomeInfoDb.Rmd:317]: I took a quick look and looks like this wasn't resolved. I am unresolving.

File: vignettes/GenomeInfoDb.Rmd (hunk 1 above)
Comment (LiNk-NY) [reply vignettes/GenomeInfoDb.Rmd:3]: It looks like it is still missing. @villafup make sure you commit to the convert-genomeinfodb branch and push to https://github.com/villafup/GenomeInfoDb/tree/convert-genomeinfodb

**Pull request closed without merging**